build.gradle

```diff
@@ -150,7 +169,7 @@ task cmakeBuild(type: Exec) {
     ]
 }
 
-// run jtreg tests. Note: needs jtreg_cmd variable set to point to the jtreg
+// run jtreg tests. Note: needs jtreg_home variable set to point to the jtreg
```

Comment:
good catch :) That comment is relic of the past.

Reply:
I am not a committer in this project. So, we've to wait for @mcimadamore for sponsoring the change anyway. I'd like him to review and sponsor. Thanks for your contribution @danieljarabek !